### DISCUSSION TITLE
What is the dashed border called that appears when selecting buttons by pressing tab? How do i change it?

### DISCUSSION THREAD

**Discussion by NoizeDaemon** [AvaloniaUI/Avalonia#11719] [Q&A] [answered]
Here's my RadioButton control, 2 got focused by checking 1 and pressing TAB once:

![image](https://github.com/AvaloniaUI/Avalonia/assets/42150677/95ac8509-856f-4647-8ffe-e0d5d8057587)
(according to the DevTools: 1 is `:checked`, 2 is `:focus`, `:focus-visible` and `:focus-within`, 3 is `:pointerover`)

In DarkMode the black dashed border is barely visible and I'd like to change the color or preferably copy the `:pointerover` effect. 
How do I do that?

I'm aware of pseudo-classes, however it's always a chore to find the correct one (do I need to use `/template/`, is it actually done in a `ContentPresenter` or some other internally nested control? etc.) - especially if you do not know what the visual is called you're trying to change 😅 I think in WPF it's `FocusVisualStyle`, but I was unable to find anything similar to that in Avalonia.

I'd appreciate any help, as well as maybe some resource that has a good overview over pseudoclasses!

Idk if that's important, but I'm using `Material.Avalonia` here.
(Bonus question related to that: How do I define different colours for light and dark mode? In general and in `Material` - altough I better ask that over at their page :P)

**Comment by jp2masa** [comment, marked as answer]
I think you're looking for `DefaultFocusAdornerProperty` (#10415)?

For light and dark modes, you can use `ThemeDictionary` (#8166).

**Comment by NoizeDaemon** [comment]
Thank you very much!

`FocusAdorner` - would have never found that name by myself 😅

```
		<Style Selector=":is(Control)">
			<Setter Property="FocusAdorner">
				<FocusAdornerTemplate>
					<Border BorderThickness="2"
							BorderBrush="{DynamicResource SecondaryHueMidBrush}">
					</Border>
				</FocusAdornerTemplate>
			</Setter>
		</Style>
```

![image](https://github.com/AvaloniaUI/Avalonia/assets/42150677/fc9cd45d-38c7-40fc-a2f7-e2da038bf2fd)

...

As for themes, I had to go a different route since I am using `Material.Avalonia`.
If someone else has a massive brain freeze like me & can't figure it out, [I asked for help](https://github.com/AvaloniaCommunity/Material.Avalonia/discussions/247) over at their repo and got it working.
